wsd/DocumentBroker.cpp

```diff
@@ -1128,48 +1123,35 @@ bool DocumentBroker::download(
     return true;
 }
 
-void DocumentBroker::lockIfEditing(const std::shared_ptr<ClientSession>& session,
-                                   const Poco::URI& uriPublic, bool userCanWrite)
+void DocumentBroker::lockIfEditing(const std::shared_ptr<ClientSession>& session)
 {
     if (_lockCtx == nullptr || !_lockCtx->supportsLocks() || _lockCtx->isLocked())
     {
         return; // Nothing to do.
     }
 
-    if (session)
```

Comment:
checking lockIfEditing wrt the dropping of "if (session)" I see that its only called from onViewLoaded and that's only called from docBroker->onViewLoaded(client_from_this()); so an empty shared_ptr scenario doesn't arise